web/src/components/Field.tsx

```diff
@@ -108,7 +104,6 @@ const Field: React.FC<IField> = ({
             to={link}
             onClick={(event) => {
               event.stopPropagation();
-              scrollTop();
             }}
```

Comment:
is <ScrollTop/> not needed here ? 👀

Reply:
nope because that was a hack I did before to make clicking on the Court value go top. with the new code it will go to top since I put the ScrollTop component in the Courts index component